modules/performing-an-image-signature-verification.adoc

```diff
@@ -0,0 +1,123 @@
+// Module included in the following assemblies:
+//
+// * operating/verify-image-signatures.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="performing-an-image-signature-verification_{context}"]
+= Performing an image signature verification
+
+Before you perform an image signature verification, you must first create a signature integration in {rh-rhacs-first}.
+
+When you create a signature integration, you can use the following verification methods:
+
+* Cosign public keys
+* Cosign certificates
+* Transparency log
+
+.Prerequisites
+
+. You have a Cosign public key which is encoded in Privacy Enhanced Mail (PEM) format.
++ 
+For more information about Cosign public keys, see link:https://docs.sigstore.dev/cosign/signing/overview/[Overview] (Sigstore documentation).
+
+. You have the certificate identity and issuer.
+. Optional: You have a certificate and chain which is encoded in PEM format. 
++
+For more information about Cosign certificates, see link:https://docs.sigstore.dev/cosign/verifying/verify/[Verifying Signatures] (Sigstore documentation).
+
+.Procedure
+
+. In the {product-title-short} portal, click *Platform Configuration* -> *Integrations*.
+. Scroll down to the *Signature Integrations* section, and click *Signature*.
+. To create a new signature integration, click *New integration*.
+. Enter a name for the integration. You can save the integration only after you specify a name for the integration.
```

Comment:
Is this actually the last required step of performing an image signature verification? I don't see any other steps for that specific procedure, but instead a long list of Optional ones. All of the Optional procedures need to be separate modules. The actual thing we are trying to do here is lost.